.github/dependabot.yml

```diff
@@ -3,17 +3,16 @@ updates:
   - package-ecosystem: "gomod"
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "monthly"
   - package-ecosystem: "npm"
     directory: "/web/ui"
-    open-pull-requests-limit: 0
     schedule:
-      interval: "weekly"
+      interval: "monthly"
```

Comment:
This will still create the problem described in https://github.com/prometheus/prometheus/pull/9973#issue-1074237580 though, right? Just on a monthly instead of weekly basis.

Reply:
well the hope here is it will avoid the minor update that is creating many PRs.
It's a last try I would say.

Reply:
Makes sense.